crypto/fipsmodule/bcm.c

```diff
@@ -408,21 +401,18 @@ int BORINGSSL_integrity_test(void) {
 
 void AWS_LC_FIPS_failure(const char* message) {
 #if defined(AWSLC_FIPS_FAILURE_CALLBACK)
-  if (AWS_LC_fips_failure_callback == NULL) {
-    fprintf(stderr, "AWS_LC_fips_failure_callback not defined but AWS-LC built with AWSLC_FIPS_FAILURE_CALLBACK. FIPS failure:\n%s", message);
-    fflush(stderr);
-    abort();
-  } else {
+  if (AWS_LC_fips_failure_callback != NULL) {
```

Comment:
We should update the [documentation here](https://github.com/aws/aws-lc/blob/dc3b44be65ac29fc1999ab2aad2d06c64f7525b6/crypto/internal.h#L1273) to indicate that a failure (even in FIPS mode) might be ignored by the callback. I.e., the failure doesn't necessarily "prevent any further cryptographic operations by the current process".

Reply:
I thought about it, I was leaving it implied because to build AWS-LC in FIPS mode you can't specify the flag to turn the callback on. But I can add something explaining this behavior.